Source/Core/Core/DSP/Interpreter/DSPIntBranch.cpp

```diff
@@ -103,13 +103,17 @@ void Interpreter::ret(const UDSPInstruction opc)
   state.pc = state.PopStack(StackRegister::Call);
 }
 
-// RTI
+// RTIcc
 // 0000 0010 1111 1111
 // Return from exception. Pops stored status register $sr from data stack
 // $st1 and program counter PC from call stack $st0 and sets $pc to this
 // location.
-void Interpreter::rti(const UDSPInstruction)
+// This instruction has a conditional form, but it is not used by any official ucode.
```

Comment:
I wonder if this is worth a game quirk report? Probably not?

Reply:
I don't think it is.  There's a pretty well-defined set of uCodes, and it's unlikely that there's an undiscovered one that uses a conditional form of RTI.  (I didn't actually look through all of the uCodes, though, since I don't have all of them on hand.)

Reply:
I have all but two ucodes and none of them use conditional rti.